dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -14,6 +14,18 @@ interface IWpnForegroundSink : IUnknown
     HRESULT InvokeAll([in] ULONG length, [in, size_is(length)] byte* data, [out] BOOL* foregroundHandled);
 };
 
+[object]
+[uuid(PUSHNOTIFICATIONS_FOREGROUNDSINK2_CLSID_UUID)]
+[pointer_default(unique)]
+interface IWpnForegroundSink2 : IUnknown
+{
+    HRESULT InvokeAllWithCorrelationVector(
+        [in] ULONG length,
+        [in, size_is(length)] byte* data,
+        [in] LPCWSTR correlationVector,
```

Comment:
Is this copied from the OS somewhere? Should this be a WinRT type instead of "flat IDL"?

Reply:
@pmpurifoy do you have context on why the sink interfaces were added here?